scraper.py

```diff
@@ -34,6 +37,7 @@ def init():
             locked BOOLEAN,
             name TEXT,
             known_candidates BIGINT,
+            sopn_published TEXT,
```

Comment:
SQLite does support a `DATETIME` type. We're using it a few lines above for the `timestamp` column.

Reply:
Annoyingly, because of the way the data gets pulled back, with the `scraperwiki.sql.select` it looks like everything is a string until you parse it into something different like we do for formatting the poll date.

Provided the code handles the None case specifically, serialising the dates in partial ISO format should be fine ordering-wise.